src/chart/parameter/ParameterSelectionChart.tsx

```diff
@@ -70,7 +72,7 @@ const NeoParameterSelectionChart = (props: ChartProps) => {
             <div style={{ width: "100%" }}>
                 <NeoField
                     key={"freetext"}
-                    label={helperText ? helperText : label + " " + property}
+                    label={helperText ? helperText : label + " " + propertyDisplay}
```

Comment:
This can also be replaced by the "||" syntax

Reply:
Done